src/Deque.mo

```diff
@@ -1,15 +1,17 @@
-/// Double-ended Queue
-///
-/// This module provides a purely-functional double-ended queue.
-
 import List "List";
 import P "Prelude";
 
+/// Functions for persistent, double-ended queues.
```

Comment:
This comment needs to be moved back to the top to appear in the synopsis for the module. Sorry, I didn't notice this during my first review.